go.sum

```diff
@@ -808,6 +808,8 @@ github.com/stretchr/testify v1.8.0/go.mod h1:yNjHg4UonilssWZ8iaSj1OCr/vHnekPRkoO
 github.com/stretchr/testify v1.8.1/go.mod h1:w2LPCIKwWwSfY2zedu0+kehJoqGctiVI29o6fzry7u4=
 github.com/stretchr/testify v1.8.2 h1:+h33VjcLVPDHtOdpUCuF+7gSuG3yGIftsP1YvFihtJ8=
 github.com/stretchr/testify v1.8.2/go.mod h1:w2LPCIKwWwSfY2zedu0+kehJoqGctiVI29o6fzry7u4=
+github.com/sttts/kube-openapi v0.0.0-20230714075721-ac6320a561bc h1:VuSO6YAOENRCSPglso/fZaGvRg7cffdsB/PCqzdo5qE=
```

Comment:
`k8s.io/kube-openapi` -> `gthub.com/sttts/kube-openapi` doesnt seem right to me

Reply:
indeed. Fixing.

Reply:
pushed